server/plugins/auth/openid.ts

```diff
@@ -0,0 +1,27 @@
+import log from "../../log";
+import Config from "../../config";
+import type {AuthHandler} from "../auth";
+
+const openIDAuth: AuthHandler = (manager, client, user, _, callback) => {
```

Comment:
nit: make this a proper function, not sure why you changed it to the inline style.
This is the top level, you don't need the scoping rules of arrow functions here